### DISCUSSION TITLE
bridge for custom op-stack chain

### DISCUSSION THREAD

**Discussion by chappie1998** [ethereum-optimism/developers#506] [Deploying the OP Stack]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

I deployed a custom op-stack chain with avail as DA. Now, I want to bridge to use the chain. but I am not able to use it. 

### Logs

![Screenshot from 2024-06-28 21-10-18](https://github.com/ethereum-optimism/developers/assets/35380717/468d698e-7ae0-4f81-92fb-18dc6bc5ffb2)
![Screenshot from 2024-06-28 21-10-31](https://github.com/ethereum-optimism/developers/assets/35380717/70eec3ae-50a6-41ed-853a-04a90650246b)

### Additional Information

Here is the bridge I am using and the transaction
https://github.com/nitantchhajed/op-stack-bridge 
https://sepolia.etherscan.io/tx/0xa7cc7bd0b8f3ff68f0dab58643769d84d93f1fd6857ac50b2e5d8ddaa98034d4

**Comment by imtipi** [comment]
can you provide `op-node` logs?

**Reply by chappie1998** [reply]
@imtipi here is op-node logs
![Screenshot from 2024-06-28 21-09-45](https://github.com/ethereum-optimism/developers/assets/35380717/4a0a9a68-259c-423f-9734-f8e24de44f4d)

**Reply by imtipi** [reply]
> @imtipi here is op-node logs ![Screenshot from 2024-06-28 21-09-45](https://private-user-images.githubusercontent.com/35380717/344440604-4a0a9a68-259c-423f-9734-f8e24de44f4d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk3NTcxMDUsIm5iZiI6MTcxOTc1NjgwNSwicGF0aCI6Ii8zNTM4MDcxNy8zNDQ0NDA2MDQtNGEwYTlhNjgtMjU5Yy00MjNmLTk3MzQtZjhlMjRkZTQ0ZjRkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjMwVDE0MTMyNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIwYjY4NTdjNTc1Y2YzYzk1Mzc5ZjE1NWQ1ZWE5MjIxOTJkNzUxYWNhNjc0NTIzNjIyNmU3MWUyNTgyYzJmOTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.cGsZA-YgM_wRkWfCC6wAqtW4xFoyRhVdIsxcDjHVXyU)

this means your opnode didn't synced,please check this https://github.com/ethereum-optimism/developers/discussions/17#discussioncomment-8570927

**Reply by chappie1998** [reply]
I am using Avail as DA. Will it still sync? After some time, I get this and keep repeating. I don't get logs like this https://github.com/ethereum-optimism/developers/discussions/17#discussioncomment-8570927
![Screenshot from 2024-06-30 20-38-45](https://github.com/ethereum-optimism/developers/assets/35380717/f3becd8c-8f39-47fc-894d-e9c30d214a07)

**Reply by imtipi** [reply]
have you seen `reconciled with L1` in opnode ?

**Reply by chappie1998** [reply]
No, just these logs keep repeating.
